### PR TITLE
Added make stuff to install the USB to UART driver...and touched gitignore

### PR DESCRIPTION
**Pull request by gvela024**
yo

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -23,9 +23,17 @@ endif
 
 ifeq ($(HOST),mac)
 install:
-	todo
-	sudo pip install esptool
-	sudo pip install nodemcu-uploader
+	echo todo
+	# sudo pip install esptool
+	# sudo pip install nodemcu-uploader
```
Comment (gvela024) [inline Makefile]: I haven't actually tried these yet so I commented them out.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (ryanplusplus) [inline .gitignore]: Nice toech!

File: Makefile [hunk 3]
```diff
@@ -23,9 +23,17 @@ endif
...
+	curl "https://www.silabs.com/documents/public/software/Mac_OSX_VCP_Driver.zip" -o "usb-to-uart-driver.zip"
+	unzip usb-to-uart-driver.zip
+	rm usb-to-uart-driver.zip
+
+	hdiutil mount SiLabsUSBDriverDisk.dmg
+	sudo installer -store -pkg "/Volumes/Silicon Labs VCP Driver Install Disk/Silicon Labs VCP Driver.pkg" -target /
+	hdiutil unmount "/Volumes/Silicon Labs VCP Driver Install Disk"
+	rm SiLabsUSBDriverDisk.dmg
```
Comment (ryanplusplus) [inline Makefile]: Thanks, this looks lit :fire:

Comment (gvela024) [inline Makefile]: Yeet

**Pull request merged**